client/src/components/Common/ChatMessageFooter.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { TextInput, View, StyleSheet, Dimensions, Platform, TouchableOpacity } from 'react-native'
+import { ChatSendButton } from './CustomButtons'
+import { Smile, Image } from "react-native-feather";
+
+interface ChatInputProps {
+    value?: string,
+    onChangeText?: (text: string) => void
+    invalid?: boolean,
+    onSend?: () => void,
+}
+
+export const ChatMessageFooter: React.FC<ChatInputProps> = ({ value, onChangeText, onSend }) => {
```

Comment:
Can you change this into ChatScreenFooter? ChatMessageFooter makes it sound like a footer for each message!